python/src/energuide/extractor.py

```diff
@@ -0,0 +1,56 @@
+import csv
+import typing
+import sys
+import cerberus
+from energuide import reader
+
+
+_SCHEMA = {
+    'EVAL_ID': {'type': 'string', 'required': True},
+    'EVAL_TYPE': {'type': 'string', 'required': True},
+
+    'HOUSEREGION': {'type': 'string', 'required': True},
+    'WEATHERLOC': {'type': 'string', 'required': True},
```

Comment:
Unless I misunderstand, I don't think the above two fields are required to be in the schema. This should be the bare minimum set: the keys needed to guarantee uniqueness, and any fields we need to drop/manipulate in the extractor itself. All other fields should be accepted and passed on.

Reply:
You are correct, momentary lapse on my part.